content/components/selectpanel.mdx

```diff
@@ -0,0 +1,222 @@
+
+Commonly, Select panel is used for metadata application to an Issue or Pull Request, like assignee and label selections, or the branch selector on a repository’s code tab.
+
+## Anatomy
+
+<img
+  width="960"
+  alt="Select panel for assigning labels to an issue. 'Apply labels to this issue' heading, close button in upper right hand corner. Search input with a magnifer icon as a visible label, listbox of label options and the first three are selected with a visual checkbox. The footer contains a button to edit labels, and cancel and apply buttons. Purple labels are overlayed onto Select panel: Overlay (dialog) pointing to the container, Search input pointing to the input, role='listbox' bracket of the listbox content, Optional action control to the button in the lower left corner, Explicit confirm / submission buttons bracket for the apply and cancel buttons in the lower right corner."
+  src="https://user-images.githubusercontent.com/40274682/228329313-7c61fff5-0f2f-4170-b319-723e8a2a3a7d.png"
+/>
+
+## Options
+
+<img
+  width="960"
+  alt="First Select panel is a branch switcher control that includes tabs, a search input, a listbox with single-select items, and a footer with an All branches link, and cancel and apply buttons. Second Select panel is titled 'Show fields, with a list of about 10 multi-select items and apply and cancel buttons in the footer."
+  src="https://user-images.githubusercontent.com/40274682/228330031-030f5dba-19c7-41e1-b4f5-73190775f777.png"
+/>
+<Caption>
+  Select panel can have as much as the branch selector component, or as little as just a multi-selection list. If the
+  contents request user input beyond menu selection, it’s likely you need a Select panel.
+</Caption>
+
+### Additional control slot
+
+<img
+  width="960"
+  alt="Three cropped Select panel footers each displaying a different option of a control in the bottom right left corner: Link, button, or checkbox input."
+  src="https://user-images.githubusercontent.com/40274682/228338168-a9424773-531e-41e3-8d95-261eb971c340.png"
+/>
+<Caption>
+  An optional control slot is available in the Select panel’s bottom left corner. Avoid use if not absolutely necessary
+  in order to avoid component and decision overwhelm.
+</Caption>
+
+### Anchoring
+
+Select panel can be anchored to the trigger, or centered within the page. If it’s centered within the page it should have a backdrop.
+
+<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+  <div>
+    <img
+      src="https://user-images.githubusercontent.com/40274682/228386690-dd175879-0dd6-4798-8fb5-b0ee096eceea.png"
+      alt=""
+    />
+    <Caption>Anchored Select panel to a trigger.</Caption>
+  </div>
+  <div>
+    <img
+      src="https://user-images.githubusercontent.com/40274682/228386689-ea191d76-3a50-41d9-98f0-9dc997caf4bf.png"
+      alt=""
+    />
+    <Caption>Select panel centered with a darker backdrop.</Caption>
+  </div>
+</Box>
+
+## Best practices
+
```

Comment:
I think some of the imagery and captions are mixed up. I'm not sure if this is a formatting issue, or something else.

Reply:
I noticed this too, but I think this may be a formatting issue, if you're using the rendered view withing GitHub.

Reply:
Yeah, I just verified in the code and it appropriately matches the way it should.